.stylelintignore

```diff
@@ -0,0 +1,3 @@
+**/node_modules
```

Comment:
Do we need another .styelintignore inside the `test/screenshot` directory or does this one handle all?

Reply:
Polite ping on this @advorak

Reply:
This root file handles all of them. IIRC `stylelint` doesn't support merging multiple `.stylelintignore` files.